models/activities/notification.go

```diff
@@ -341,6 +345,90 @@ func GetIssueNotification(ctx context.Context, userID, issueID int64) (*Notifica
 	return notification, err
 }
 
+// CreateOrUpdateIssueNotifications creates an release notification
+// for each watcher, or updates it if already exists
+func CreateOrUpdateReleaseNotifications(releaseID, receiverID int64) error {
```

Comment:
Use `ctx context.Context` as the first parameter.

Reply:
This is called from a Queue, so we don't have a context